packages/idafree.vm/tools/chocolateyinstall.ps1

```diff
@@ -30,16 +30,23 @@ try {
     Remove-Item $desktopShortcut -Force -ea 0
   }
 
+  # Download ida_launcher.exe to assist with taskbar and right click option and store it in %RAW_TOOLS_DIR%
+  # ida_launcher.exe is a custom binary that searches for the latest ida64.exe and executes it
+  $launcherName = 'ida_launcher'
+  $launcherSource = 'https://raw.githubusercontent.com/mandiant/vm-packages/main/ida_launcher/ida_launcher.exe'
+  $launcherPath = Join-Path ${Env:RAW_TOOLS_DIR} "$launcherName.exe"
+  Write-Host "[+] Downloading '$launcherSource'"
+  (New-Object System.Net.WebClient).DownloadFile($launcherSource, $launcherPath)
```

Comment:
I think it would be better to use Chocolatey function to download files. I think they would allow us to check the hash of the binary.